de/main.po

```diff
@@ -1,815 +1,881 @@
+"dem **[offiziellen Server](%s)** bei!"
 
 msgid "Valid Example"
-msgstr ""
+msgstr "Gültiges Beispiel"
 
 msgid "Valid Examples"
-msgstr ""
+msgstr "Gültige Beispiele"
 
 msgid "For a full list of supported languages, [click here](%s)."
-msgstr ""
+msgstr "[Klicke hier](%s) für eine volle Liste aller unterstützten Sprachen."
 
 msgid "If you need help with that, you can visit the **[support server](%s)**."
 msgstr ""
+"Wenn du damit Hilfe brauchst, kannst du unseren **[Support-Server](%s)** "
+"besuchen."
 
 msgid "Do you want it back? Share your opinion at the **[support server](%s)**."
 msgstr ""
+"Du willst das zurückhaben? Teile deine Meinung in unserem **[Support-Server]"
+"(%s)**."
 
 msgid "Mods/Admins Command"
-msgstr ""
+msgstr "Mods/Admins Befehl"
 
 msgid "Admins Only"
-msgstr ""
+msgstr "Nur Admins"
 
 msgid "Change  **%s**  to  **%s**"
-msgstr ""
+msgstr "Ändere **%s** zu **%s**"
 
 msgid "Groups Quota"
-msgstr ""
+msgstr "Gruppen-Quote"
 
 msgid "More info on our **[website](%s)**, or the **[support server](%s)**."
-msgstr ""
+msgstr "Mehr Informationen auf unserer **[Webseite](%s)**."
 
 msgid "Linked Channels per Group Quota"
-msgstr ""
+msgstr "Verlinkte Kanäle pro Gruppen-Quote"
 
 msgid "Server Quota"
-msgstr ""
+msgstr "Server-Quote"
 
 msgid "Channel Information"
-msgstr ""
+msgstr "Kanal-Information"
 
 msgid "Language"
-msgstr ""
+msgstr "Sprache"
 
 msgid "Text Style"
-msgstr ""
+msgstr "Text-Stil"
 
 msgid "Translations"
-msgstr ""
+msgstr "Übersetzungen"
 
 msgid "messages"
-msgstr ""
+msgstr "Nachrichten"
 
 msgid "Automatic Translation"
-msgstr ""
+msgstr "Automatische Übersetzung"
 
 msgid "Auto Group"
-msgstr ""
+msgstr "Auto-Gruppe"
 
 msgid "Auto Groups"
-msgstr ""
+msgstr "Auto-Gruppen"
 
 msgid "Auto Mode"
-msgstr ""
+msgstr "Auto-Modus"
 
 msgid "no change"
-msgstr ""
+msgstr "keine Änderungen"
 
 msgid "To change language, type:"
-msgstr ""
+msgstr "Um die Sprache zu ändern, tippe:"
 
 msgid "Example:"
-msgstr ""
+msgstr "Beispiel:"
 
 msgid "Linked Channels in Group"
-msgstr ""
+msgstr "Verlinkte Kanäle in der Gruppe"
 
 msgid "To change group, type:"
-msgstr ""
+msgstr "Um die Gruppe zu ändern, tippe:"
 
 msgid "To change mode, type:"
-msgstr ""
+msgstr "Um den Modus zu ändern, tippe:"
 
 msgid "To change style, type:"
-msgstr ""
+msgstr "Um den Stil zu ändern, tippe:"
 
 msgid "%s's text style has been updated to **%s**."
-msgstr ""
+msgstr "%ss Text-Stil wurde zu **%s** geändert."
 
 msgid "Automatic group **%s** has been deleted."
-msgstr ""
+msgstr "Automatische Gruppe **%s** wurde gelöscht."
 
 msgid ""
 "Discord Translator is the most powerful translation bot for Discord.\n"
 "Based on *Google Cloud Translate* and *Discord.js*."
 msgstr ""
+"Discord Translator ist der mächtigste Übersetzungs-Bot für Discord.\n"
+"Basiert auf *Google Cloud Translate* und *Discord.js*."
 
 msgid "Read Wiki"
-msgstr ""
+msgstr "Wiki lesen"
 
 msgid "Tech support server"
-msgstr ""
+msgstr "Tech Support Server"
```

Comment:
Not sure about this one. Is `Tech` just a short form or is it used as a real word? Then for German everything could be one word.

Reply:
How is `Tech Support` usually referred to in German?